app/src/hooks/connectedWeb3.tsx

```diff
@@ -31,16 +31,20 @@ export const useConnectedWeb3Context = () => {
 
   return context
 }
-
+interface Props {
+  children?: React.ReactNode
+  setStatus?: any
+}
 /**
  * Component used to render components that depend on Web3 being available. These components can then
  * `useConnectedWeb3Context` safely to get web3 stuff without having to null check it.
  */
-export const ConnectedWeb3: React.FC = props => {
+export const ConnectedWeb3: React.FC<Props> = (props: Props) => {
   const [networkId, setNetworkId] = useState<number | null>(null)
   const safeAppInfo = useSafeApp()
   const context = useWeb3Context()
   const { account, active, error, library } = context
+  const rpcAddress: string | null = sessionStorage.getItem('rpcAddress')
```

Comment:
Did you consider the usage of sessionStorage vs localStorage? 
Do you want the user settings to carry over if the user closes the tab and opens again?
If so better to use localStorage.

Reply:
yeah i tried localStorage but in my opinion its too persistent...whats your opinion? I think its okay when user closes tab it goes away...

Reply:
I personally think that it should be localStorage as well, since changing the rpc URL is an infrequent action and I'd want my setting to be persisted if I close the tab.